src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -69,13 +71,43 @@ public void onEnable() {
 
 		// Log when plugin is fully enabled ( setup complete ).
 		getLogger().info( this.getLang( "log.enabled" ) );
-		//check bungeecord mode/offline mode
-		if (!Bukkit.getOnlineMode() && !Bukkit.spigot().getConfig().getBoolean("settings.bungeecord")) {
-			getLogger().severe(String.valueOf(Bukkit.spigot().getConfig().getBoolean("settings.bungeecord")));
-			getLogger().severe("WooMinecraft doesn't support offLine mode");
-			Bukkit.getPluginManager().disablePlugin(this);
+		//check bungeecord/velocity mode/offline mode
+		try {
+			getLogger().info(Bukkit.getServer().getName());

```

Comment:
Repeating usages of `Bukkit.getServer().getName()` let's make this a variable. Keep the code `DRY`